backend/user/src/express-app.js

```diff
@@ -1,25 +1,20 @@
-const express = require("express");
-const cors = require("cors");
-const { user, appEvents } = require("./api");
-const { CreateChannel, SubscribeMessage } = require("./utils");
-const cookieParser = require("cookie-parser");
-
-module.exports = async app => {
-	app.use(express.json());
-	app.use(
-		cors({
-			origin: ["http://localhost:3000"],
-			methods: ["GET", "POST", "PUT", "DELETE"],
-			credentials: true,
-		})
-	);
-	app.use(express.static(__dirname + "/public"));
-	app.use(cookieParser());
-
-	//api
-	// appEvents(app);
-
-	const channel = await CreateChannel();
-
-	user(app, channel);
-};
+const express = require("express");
+const cors = require("cors");
+const { user, appEvents } = require("./api");
+const { CreateChannel, SubscribeMessage } = require("./utils");
+
+module.exports = async (app) => {
+	app.use(express.json());
+	app.use(
+		cors({
+			origin: ["http://localhost:3000"],

```

Comment:
Same here?

Reply:
It depends on the loadbalancer IP which changes evrytime we do terraform apply/destroy. But as I said I don't really think cors is a problem in production only in developement.